Client/Client.Qt/Application.cpp

```diff
@@ -56,7 +56,8 @@ void Application::setAppState(App::AppState app_state)
         }
         case App::AppState::ChatWindowForm:
         {
-            _mainwidget->addWidget(std::make_unique<ChatWindow>());
+            if (_mainwidget->getCountWidgets() < 3)
```

Comment:
What is it for?

Reply:
This is necessary to create only one chat window